File: BasicAppSample/app/src/javaVersion/java/com/example/basicappsample/MainActivity.java
package com.example.basicappsample;

import android.os.Bundle;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.basicappsample.helpers.AnimationHelper;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView textHello = findViewById(R.id.text_hello);
        textHello.setText("Hello, Android Java");
        AnimationHelper.startAlphaAnimation(textHello);
    }
}

File: BasicAppSample/app/src/kotlinVersion/java/com/example/basicappsample/MainActivity.kt
package com.example.basicappsample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.example.basicappsample.helpers.AnimationHelper

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val textHello: TextView = findViewById(R.id.text_hello)
        textHello.text = "Hello, Android Kotlin"
        AnimationHelper.startAlphaAnimation(textHello)
    }
}